keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -277,65 +269,73 @@ the `/resize` subresource:
 To provide fine-grained user control, PodSpec.Containers is extended with
 ResizeRestartPolicy - a list of named subobjects (new object) that supports
 'cpu' and 'memory' as names. It supports the following restart policy values:
-* NotRequired - default value; resize the Container without restart, if possible.
-* RestartContainer - the container requires a restart to apply new resource values.
+* `PreferNoRestart` - default value; resize the Container without restart, if possible.
```

Comment:
Can we clarify what if possible means here? Whether it will restart anyways or not take action or will be retried later indefinitely?

Reply:
This is going to depend somewhat on what we decide to do with memory limit decreases. I'll add a bit more detail though.